fedora-spec-files/Makefile

```diff
@@ -0,0 +1,18 @@
+IMAGE_NAME := fedora-spec-research
+PODMAN_RUN_THIS_DIR := podman run --rm -v $(CURDIR):/fedora-spec-files:Z -w /fedora-spec-files
+
+# build the container image with the jupyter notebook and all the dependencies
+build:
+	podman build --tag=$(IMAGE_NAME) .
+
+# run jupyter notebook
+run:
+	@# the rooless user doesn't have access to the folder
+	@chmod -R +rwx .
+	$(PODMAN_RUN_THIS_DIR) --net=host -it $(IMAGE_NAME) jupyter notebook --allow-root ./fedora-spec-files.ipynb
+
+# Run the data collection tooling and generate data.json
+generate-data:
+	@# the rooless user doesn't have access to the folder
+	@chmod -R +rwx .
```

Comment:
Things should work without this. The "rooless" user has access to all the volume mounts done with the `Z` flag.